doc/content/resources/neams-workbench.md

```diff
@@ -0,0 +1,162 @@
+
+!media neams_workbench/demonstration_video_youtoube.png style=width:80% id=fig:demonstration_video_youtube caption=Demonstration video on how to use the NEAMS Workbench GUI on INL HPC platform.
+
+Note that a user may experience some normal delays between their click and the NEAMS Workbench GUI responding, which is due to OnDemand mechanics. If a user needs assistance with running a case or has any questions, feel free to reach out to Robert Lefebvre ([lefebvrera@ornl.gov](mailto:lefebvrera@ornl.gov)) or Marco Delchini ([delchinimg@ornl.gov](mailto:delchinimg@ornl.gov)).
+
+# The NEAMS Workbench
+
+[The NEAMS Workbench](https://www.ornl.gov/onramp/neams-workbench) is an open-source licensed graphical user interface ([GUI](https://code.ornl.gov/neams-workbench/downloads)) developed at [Oak Ridge National Laboratory](https://www.ornl.gov/) that integrates [ParaView](https://www.paraview.org/) for visualizing the numerical solution. In recent years, Workbench has integrated MOOSE-based applications, among others. The NEAMS Workbench provides a user-friendly interface for creating, editing, and validating input files to be run on desktop computers or HPC platforms and for visualizing the numerical solution with the built-in ParaView visualization and analysis software. Its Python-based application runtime environment, template engine, and domain-specific language processing tool enables the integration of a wide range of applications and advanced workflows.
+
+# Cloning the VTB GitHub repository
+
+Before running the NEAMS Workbench GUI on the INL HPC platform, it is recommended to clone the VTB GitHub repository to your home directory on Sawtooth to access the input files used in the subsequent sections. This can be achieved from the [HPC OnDemand website](https://hpcondemand.inl.gov/webauthentication) by starting a Sawtooth terminal as shown in [fig:hpc_on_demand_sawtooth_terminal].
+
+!media neams_workbench/hpc_on_demand_sawtooth_terminal.png style=width:80% id=fig:hpc_on_demand_sawtooth_terminal caption=Accessing Sawtooth terminal on the INL HPC OnDemand website.
+
+The Sawtooth terminal should immediately start as seen in [fig:sawtooth_terminal]:
+
+!media neams_workbench/sawtooth_terminal.png style=width:80% id=fig:sawtooth_terminal caption=Sawtooth terminal on the INL HPC OnDemand website.
+
+Once the terminal is opened, the VTB GitHub repository can be cloned. The first step is to load the `gitlfs` module and to install `Git LFS` locally. [Git-LFS](https://git-lfs.github.com/) is needed to clone large size files (mesh, database, ...) present in the VTB GitHub repository:
+
+```console
+module load gitlfs; git lfs install
+```
```

Comment:
move this in a separate section and have a caveat that it depends on the model. We actually dont need lfs for most inputs
The list of inputs needing it:
```
pbfhr/reflector/conduction/conduction.fld filter=lfs diff=lfs merge=lfs -text
pbfhr/reflector/meshes/solid.e filter=lfs diff=lfs merge=lfs -text
pbfhr/reflector/meshes/fluid.re2 filter=lfs diff=lfs merge=lfs -text
msfr/steady/sample_output/run_neutronics_out.e filter=lfs diff=lfs merge=lfs -text
msfr/steady/sample_output/run_neutronics_out_ns0.e filter=lfs diff=lfs merge=lfs -text
mrad/mesh/mrad_mesh.e filter=lfs diff=lfs merge=lfs -text
htgr/pbmr400/shared/oecd_pbmr400_tabulated_xs.xml filter=lfs diff=lfs merge=lfs -text
```

Reply:
so basically a need only for Nek meshes, mrad mesh and an HTGR cross section file

Reply:
I remember getting an error message when cloning the VTB repo if `git-lfs` was not available. So what is the correct workflow and command lines to close the VTB repo?

Reply:
just clone unless you need one of these files

Reply:
I think we ll want to have a paragraph about LFS somewhere else and link to it in every input that needs it?

Reply:
I would rather leave it as it is. The instructions on the VTB website on how to close the VTB repo requires the user to install `git-lfs`. These instructions are specific to Sawtooth.